app/Jobs/StatusPipeline/RemoteStatusDelete.php

```diff
@@ -153,7 +153,10 @@ public function unlinkRemoveMedia($status)
             ->whereObjectId($status->id)
             ->delete();
         StatusArchived::whereStatusId($status->id)->delete();
-        StatusHashtag::whereStatusId($status->id)->delete();
+        $statusHashtags = StatusHashtag::whereStatusId($status->id)->get();
+        foreach($statusHashtags as $stag) {
+        	$stag->delete();
+        }
```

Comment:
Can't this be done by telling laravel to cascade the deletion?

Reply:
See: https://laravel.com/docs/10.x/migrations#foreign-key-constraints

Reply:
This isn't about cascading deletes or the database, rather we are doing this to trigger the deleting/deleted events for the StatusHashtagObserver to fanout create/deletes!